paper-api/src/main/java/org/bukkit/Bukkit.java

```diff
@@ -1373,7 +1373,10 @@ public static int getSpawnRadius() {
      * Sets the radius, in blocks, around each worlds spawn point to protect.
      *
      * @param value new spawn radius, or 0 if none
+     * @deprecated don't work anymore, set the {@code spawn-protection} in
```

Comment:
`does not work` feels better than `don't work anymore` which reads as a command to stop working.

Reply:
The deprecation message is too user focused, this is a dev thing